std_msgs/Int32 maxFeatures
std_msgs/Float32 scale
std_msgs/Int32 edge
std_msgs/Int32 level
std_msgs/Int32 wta
std_msgs/Int32 score
std_msgs/Int32 patch
